glib/src/auto/functions.rs

```diff
@@ -196,6 +196,73 @@ pub fn dpgettext2(domain: Option<&str>, context: &str, msgid: &str) -> crate::GS
     }
 }
 
+#[doc(alias = "g_environ_getenv")]
+pub fn environ_getenv(
+    envp: &[&std::path::Path],
+    variable: impl AsRef<std::path::Path>,
```

Comment:
All the `env`-style functions should work with `OsStr` / `OsString` instead of `Path`/`PathBuf`. I think there is `Gir.toml` configuration for that: `string_type = "os_string"`